provider_sources/xyzservices-providers.json

```
@@ -40,6 +40,27 @@
             "html_attribution": "Imagery provided by services from the Global Imagery Browse Services (GIBS), operated by the NASA/GSFC/Earth Science Data and Information System (<a href=\"https://earthdata.nasa.gov\">ESDIS</a>) with funding provided by NASA/HQ.",
             "name": "NASAGIBS.ViirsTrueColorCR",
             "time": ""
+        },
+        "BlueMarble3413": {
+            "url": "https://gibs.earthdata.nasa.gov/wmts/epsg3413/best/BlueMarble_NextGeneration/default/EPSG3413_500m/{z}/{y}/{x}.jpeg",
+            "max_zoom": 5,
+            "attribution": "Imagery provided by services from the Global Imagery Browse Services (GIBS), operated by the NASA/GSFC/Earth Science Data and Information System (<a href=\"https://earthdata.nasa.gov\">ESDIS</a>) with funding provided by NASA/HQ.",
+            "html_attribution": "Imagery provided by services from the Global Imagery Browse Services (GIBS), operated by the NASA/GSFC/Earth Science Data and Information System (<a href=\"https://earthdata.nasa.gov\">ESDIS</a>) with funding provided by NASA/HQ.",
+            "name": "NASAGIBS.BlueMarble3413"
```

```suggestion
            "name": "NASAGIBS.BlueMarble3413",
            "crs": "EPSG:3413"
```

Since these are not in expected 3857, I think it is better to explicitly state that.